test/suite/test_tiered04.py

```diff
@@ -235,8 +237,18 @@ def test_tiered(self):
         self.check_metadata(self.tiereduri, last)
         self.check_metadata(self.tiereduri, oldest)
         self.check_metadata(fileuri, intl_page)
+        self.check_metadata(fileuri, intl_page)
```

Comment:
These two lines are the same. Is this an error, or is there something subtle going on here?

Reply:
Good catch. It was a copy/paste error. Fixed.